test/hotspot/jtreg/runtime/ErrorHandling/MachCodeFramesInErrorFile.java

```diff
@@ -180,6 +186,6 @@ private static void extractFrames(String hsErr, Set<String> frames, boolean nati
                 frames.add(line);
             }
         }
-        throw new RuntimeException("\"" + marker + "\" line missing in hs_err_pid file:\n" + hsErr);
+        throw new RuntimeException("\"" + marker + "\" line missing in hs_err_pid file:\n");
```

Comment:
Please explain!

Reply:
OK!
This was an odd one; I was getting errors that complained about the length of the message in the exception. It looked like 2^16 was the max number of characters that could be included in a RuntimeException. I haven't found the upper limit documented anywhere, but I would get unexpected failures with it included in the message. The text prints to stdout, so I didn't suffer by removing it.

Reply:
Maybe just print the last line?

Reply:
I did a bit more testing to jog my memory (it has been a little while since I made these changes). You were right to be concerned about the removal; the error text is not printed automatically.

To avoid having an exception message be too long, I printed the String to std[out|err] using the usual System.[out|err].println methods.

Good catch :slightly_smiling_face: